deploy/action.yml

```diff
@@ -2,75 +2,92 @@ name: Deploy Storefront
 description: Builds Middleware app docker image
 inputs:
   project_name:
-    description: 'Project name'
+    description: "Project name"
     required: true
   docker_registry_url:
-    description: 'Docker registry url'
+    description: "Docker registry url"
     required: false
-    default: 'registry.vuestorefront.cloud'
+    default: "registry.vuestorefront.cloud"
   cloud_username:
-    description: 'Cloud username'
+    description: "Cloud username"
     required: true
   cloud_password:
-    description: 'Cloud password'
+    description: "Cloud password"
     required: true
   cloud_region:
-    description: 'Cloud region'
+    description: "Cloud region"
     required: true
   console_api_url:
-    description: 'URI for Vue Storefront Console API'
+    description: "URI for Vue Storefront Console API"
     required: false
-    default: 'https://api.platform.vuestorefront.io'
+    default: "https://api.platform.vuestorefront.io"
   version:
-    description: 'Version of the app'
+    description: "Version of the app"
     required: false
 
 runs:
   using: "composite"
   steps:
-    - uses: chrnorm/deployment-action@releases/v2
+    - uses: chrnorm/deployment-action@v2
       name: Create GitHub deployment
       id: deployment
       with:
-        token: '${{ github.token }}'
+        token: "${{ github.token }}"
         environment-url: https://${{ inputs.project_name }}.${{ inputs.cloud_region }}.gcp.storefrontcloud.io
-        environment: production
         initial-status: in_progress
-
-    - name: Deploy Frontend App (${{ inputs.project_name }}.${{ inputs.cloud_region }}.gcp.storefrontcloud.io)
-      uses: fjogeleit/http-request-action@v1
-      with:
-        url: "${{ inputs.console_api_url }}/cloud/instances/${{ inputs.project_name }}-${{ inputs.cloud_region }}-gcp-storefrontcloud-io/deploy"
-        method: "PATCH"
-        customHeaders: '{"Content-Type":"application/json"}'
-        data: '{ "cloudUserId":"${{ inputs.cloud_username }}", "cloudUserPassword":"${{ inputs.cloud_password }}", "dockerImageHash":"${{ inputs.version || github.sha }}" }'
-        timeout: 60000
-
-    - name: Deploy Middleware (${{ inputs.project_name }}.${{ inputs.cloud_region }}.gcp.storefrontcloud.io)
-      uses: fjogeleit/http-request-action@v1
-      with:
-        url: "${{ inputs.console_api_url }}/cloud/instances/${{ inputs.project_name }}-${{ inputs.cloud_region }}-gcp-storefrontcloud-io/deploy"
-        method: "PATCH"
-        customHeaders: '{"Content-Type":"application/json"}'
-        data: '{ "middleware":true, "cloudUserId":"${{ inputs.cloud_username }}", "cloudUserPassword":"${{ inputs.cloud_password }}", "dockerImageHash":"${{ inputs.version || github.sha }}", "middlewareData":{ "path":"/api/", "port":4000, "has_base_path":false } }'
-        timeout: 60000
+    - name: Deploy on ${{ inputs.project_name }}.${{ inputs.cloud_region }}.gcp.storefrontcloud.io
```

Comment:
i changed this because this code is what the SA's are familiar with. What do you think?

Reply:
Deployments must be triggered via the Console API. The firing it through the Farmer is a deprecated way.

Reply:
Triggering it via Farmer, erases the environment variables and there's lack of deployment markers in the Console — so part of their functionality is broken.